docs/contributing/contributing.rst

```diff
@@ -310,6 +311,31 @@ with ``worker`` to get shell access in your worker.
     if Docker can't find the file/directory on the host it will silently fail and mount
     in an empty directory.
 
+
+.. admonition:: Enabling FUSE
+
+    When running toil-wdl-runner with Singularity, Singularity will sandbox images
+    by default. This can take time if a workflow has lots of images. To avoid this, FUSE can be given
+    to the Docker container at startup. There are 2 main ways to do this. Either run all the Docker
```

Comment:
FUSE is a whole project/kernel feature; I think it sounds odd to say that we give FUSE itself to the container.
```suggestion
    by default. This can take time if a workflow has lots of images. To avoid this, access to FUSE can be given
    to the Docker container at startup. There are 2 main ways to do this. Either run all the Docker
```